src/test/ui/drop/drop_order.rs

```diff
@@ -1,4 +1,5 @@
 // run-pass
```

Comment:
Could you add MIR validation to this test, like e.g. done [here](https://github.com/rust-lang/rust/pull/102394/files#diff-5f549293fd7dba2a7e7429b9513d703e0c3f615d49e5369623a31c3a73725caaR3)? There have been errors for let else related to invalid MIR, better to test for it.

Reply:
Done!